README.md

```diff
@@ -1,2 +1,10 @@
 # tdex-app
 📱 Mobile app for making trades on TDEX 
+
+## Run
+
+1. Clone repo
+
+2. ```sh  $ npm install```
+
+3. ```sh $ npm run start``` it opens it in browser
```

Comment:
It's missing how to create a build. (wich on my computer fails with 

```
./src/App.tsx
Cannot find file './pages/Tabs' in './src'.

```

Are you sure we dont' need to inform the developer how to build the scss too?

Reply:
it fails cause i think i need add all scss, my bad

Reply:
@tiero I think now it should work, it was bad idea maybe to not split code for small prs while i was doing markup